### ISSUE TITLE
Sass compile errors

### ISSUE THREAD

**Issue by fjarrett** [godaddy/wp-ascension-theme#45]
@schrapel Every time I try to run `grunt sass` I get compile errors like this one:

```
Running "sass:dist" (sass) task
>> Error: "#content" failed to @extend ".row".
>>        The selector ".row" was not found.
>>        Use "@extend .row !optional" if the extend should be able to fail.
>>         on line 8 of .dev/sass/partials/_layout.scss
>> >> 	@extend .row;
>>    -^
Warning:  Use --force to continue.
```

Am I supposed to be running an older version of `grunt-sass`? I'm currently running `^1.2.0`.

Why would the grid selectors not be available?

**Comment by schrapel** [comment]
Yes, you need to use an older version. @chriswallace knows which version. I
am not exactly sure what the issue was but downgrading solved it for me
too. It probably has to do with the grid system and we need to include not
extend here.

I use nvm so I can switch between versions specifically for this problem

**Comment by chriswallace** [comment]
I'm looking at this today.

**Comment by chriswallace** [comment]
This is fixed in 7afc4fa3fcf5c08afbb25bb4e16ed86e308667a0 and I also updated Gruntfile.js and included the devtools package in fcda8a8fe841676952f44f469f42eeb61f39e4d3.
